src/components/Statistics/StatisticCardAll.tsx

```diff
@@ -33,7 +39,7 @@ export const StatisticCardAll: React.FC<IProp> = ({ data, debtCurrent, collatera
         <Grid id='debt' item xs={12} sm={5}>
           <StatisticsCard
             name='Debt'
-            value={debtCurrent.reduce((a, b) => a + b.value, 0)}
+            value={debtCurrent.reduce((a, b) => a + b.value, 0) + debtVault}
```

Comment:
I talked about that with Norbert and he said that nothing should be added to value of total debt (which is the value of minted synthetics and it already grows when you mint something through staking or swapline)

Reply:
ok, I'll delete it